### PR TITLE
ci: Sonar scanner update

### PR DESCRIPTION
**Pull request by jirimosinger**

### CODE REVIEW

File: .github/workflows/Sonarcloud-scan.yml [hunk 1]
```diff
@@ -70,19 +70,13 @@ jobs:
...
           unzip sonar-scanner.zip
           mv sonar-scanner-$SONAR_SCANNER_VERSION-linux sonar-scanner-linux
         working-directory: ${{ runner.temp }}
 
-      - name: Set up JDK 17
-        uses: actions/setup-java@v3
-        with:
-          java-version: 17
-          distribution: 'temurin'
-
```
Comment (slavek-kucera) [inline .github/workflows/Sonarcloud-scan.yml:85]: Can we actually remove it?

Comment (slavek-kucera) [reply .github/workflows/Sonarcloud-scan.yml:85]: The 4.2 packed its own JRE, the 5.0 does not and GH Actions default to java11

Comment (slavek-kucera) [reply .github/workflows/Sonarcloud-scan.yml:85]: ok, no problem, each package is done differently.... the -linux variant has its jre...

**Review by slavek-kucera — APPROVED** [review]

File: .github/workflows/Sonarcloud-scan.yml (hunk 1 above)
Comment (jirimosinger) [reply .github/workflows/Sonarcloud-scan.yml:85]: Seems like it. I was testing it using docker where I've had only version 11, nevertheless, I see that it prints out the following:

```
...
INFO: Scanner configuration file: /sonar-scanner-linux/conf/sonar-scanner.properties
INFO: Project root configuration file: NONE
INFO: SonarScanner 5.0.1.3006
INFO: Java 17.0.7 Eclipse Adoptium (64-bit)
INFO: Linux 5.15.79.1-microsoft-standard-WSL2 amd64
INFO: SONAR_SCANNER_OPTS=-server
INFO: User cache: /root/.sonar/cache
...
```

**Pull request merged**